### ISSUE TITLE
Operator 'plot' doesn't work if field type is 'short'

### ISSUE THREAD

**Issue by senesis** [senesis/climaf#30]

**Comment by jservonnat** [comment]
Following this ncl tutorial web page (https://www.ncl.ucar.edu/Document/Functions/Contributed/short2flt.shtml) the solution is to convert the field from short to float ; at the same time it takes into account the attributes "add_offset" and "scale_factor". It does the conversion that was missing in the plots.
Here are the code lines for gplot.ncl and plotmap.ncl:
- change this:
```
  ; Reading the field and its unit, setting REDU, and scaling the field
  if isfilepresent(infile) then
    ffile=addfile(infile,"r")
    if isfilevar(ffile,var) then fld = ffile->$var$
    else
      print("File "+infile+" does not include variable "+var)
      status_exit(1)
    end if
  else
    print("File "+infile+" cannot be accessed")
    status_exit(2)
  end if
```
- for this:
```
  ; Reading the field and its unit, setting REDU, and scaling the field
  if isfilepresent(infile) then
     ffile=addfile(infile,"r")
     if isfilevar(ffile,var) then
        if (getfilevartypes (ffile,var).eq."short") then
            print("Found that it is a short")
            fld  = short2flt( ffile->$var$ )
        else
            fld = ffile->$var$
        end if
     else
        print("File "+infile+" does not include variable "+var)
        status_exit(1)
     end if
  else
    print("File "+infile+" cannot be accessed")
    status_exit(2)
  end if
```

**Comment by jservonnat** [comment]
However, we will need to be sure that this issue with the short format and the "scale_factor" and "add_offset" attributes won't be a problem elsewhere.

**Comment by LudivineV** [comment]
Ok, it will be done

**Comment by senesis** [comment]
fixed in branch dev  : 3283a85
